README.md

```diff
@@ -97,6 +97,14 @@ using different configurations. Check the `./getting-started` directory for more
 - `kubectl get deployment -n polaris` - To check the status of the deployment.
 - `kubectl describe deployment polaris-deployment -n polaris` - To troubleshoot if things aren't working as expected.
 
+#### Configuring Polaris
+
+Polaris Servers can be configured using a variety of ways.
+Please see the [Configuration Guide](https://polaris.apache.org/in-dev/unreleased/configuration/)
```

Comment:
Can we use relative path here as well?

Reply:
I'm not sure. I suppose people often view this README on GH or locally.

Reply:
I have test a relative path as the following. Either GH and local(in IDE) usage are fine. Just a bit concern of inconsistency, as once we got release. The `https://polaris.apache.org/in-dev/unreleased/configuration/` will be changed. 
```
- See [README in `site/`](site/README.md) for more information.
```

Reply:
Changed to a relative link to `configuration.md`. This is a link between `.md` files, not to the generated docs, but that works for me.